Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -1769,7 +1769,8 @@ void HTMLMediaElement::loadResource(const URL& initialURL, const ContentType& in
 {
     ASSERT(initialURL.isEmpty() || isSafeToLoadURL(initialURL, InvalidURLAction::Complain));
 
-    INFO_LOG(LOGIDENTIFIER, initialURL, initialContentType);
+    auto logSiteIdentifier = LOGIDENTIFIER;
```

Comment:
All the changes replacing the INFO_LOG(LOGIDENTIFIER) with INFO_LOG(logSiteIdentifier) seems unrelated. Are these needed here ?

If not, can we take this to a different change, or describe them in the changeling ?

Reply:
They are needed so logging from inside the lambda includes the method name